### PR TITLE
Dict: Unneeded

### PR DESCRIPTION
**Pull request by sebweb3r**
I'm not sure about this one. 

Cambridge doesn't accept unneeded
https://dictionary.cambridge.org/spellcheck/english/?q=unneeded
However aspell does
http://app.aspell.net/lookup?dict=en_US;words=unneeded

Whats the opinion of our natives @lurch @peternewman

### CODE REVIEW

**Comment by lurch** [comment]
It does sound a bit odd. I think I'd suggest `unneeded->unnecessary, not needed,` ?

**Comment by peternewman** [comment]
The GB large aspell has it too:
http://app.aspell.net/lookup?dict=en_GB-large;words=unneeded

It's in Collins too:
https://www.collinsdictionary.com/dictionary/english/unneeded

So I think it's a usage/style issue at best.

**Comment by sebweb3r** [comment]
So how to proceed?
Put it in informal or usage? Or close the PR?

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -28879,7 +28879,7 @@ uneccesary->unnecessary
 uneccessarily->unnecessarily
 unecessarily->unnecessarily
 unecessary->unnecessary
-uneeded->unneeded, needed,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:28882]: Suggest unheeded too

**Review by peternewman — COMMENTED** [review]
Another suggestion for the original

**Comment by peternewman** [comment]
> So how to proceed?
> Put it in informal or usage? Or close the PR?

Up to you, informal would be the right choice if at all I think, usage was the wrong one. I'm not convinced how informal it is personally.

I did just find one improvement to the existing entries though...

**Comment by sebweb3r** [comment]
I will put it in my local dictionary. I created a new MR for your suggestion to not flood the git history.

**Comment by peternewman** [comment]
> I will put it in my local dictionary. I created a new MR for your suggestion to not flood the git history.

Fair  enough, thanks for that.

I didn't realise a PR could close another!

**Pull request closed without merging**